pyirf/plot.py

```diff
@@ -0,0 +1,36 @@
+import matplotlib.pyplot as plt
+from astropy.io import fits
+
+
+def plot_angular_resolution(irf_filename, ax=None, **kwargs):
```

Comment:
I would use our in memory objects as input, not a filename.

Reply:
right!

Reply:
so give it `true_energy_bins` and `angular_resolution`.

Reply:
so that would be basically what exists already in `ctaplot`
I have no problem copy/pasting code - just to know what direction to take

Reply:
If a function in ctaplot already exists, great. I would just use it then. Not duplicate the code.

I am not to keen to have a lot of plotting code in here.

Reply:
Ok but we agree to have a plotting module anyway to aggregate the existing implementation (ctaplot or gammapy) directly here?

Reply:
We could add that yes. Or just add an example script using either of those.

Providing wrappers around gammapy or ctaplot in `pyirf/plot` seems a bit unncessary to me, but if other's think it would be good to have, I am not against it.

Reply:
My point of view summarized:
- we should have a plotting module for internal fast check of the results without needing to _know_ gammapy
- no need to re-invent the wheel, so we could either import functions from _ctaplot_ or from _gammapy_
- if we diverge from (widely used?) packages like _gammapy_, let's make sure that we provide the same result